src/discretize.jl

```diff
@@ -586,8 +585,24 @@ function SciMLBase.symbolic_discretize(pde_system::PDESystem,
                                                            pde_loss_functions,
                                                            bc_loss_functions)
 
-    function full_loss_function(θ, p)
+    if bayesian
+        # required as Physics loss also needed on dataset domain points
+        pde_loss_functions1, bc_loss_functions1 = if !(dataset_given isa Nothing)
+            if !(strategy isa  GridTraining)
```

Comment:
why would only gridtraining be allowed? Wouldn't you just add these points?

Reply:
yeah, the points would get included into the training sets, I end up using the GridTraining strategy for loss on those points anyways so was thinking i should allow GridTraining as then throughout domain you have the same strategy applied